laura/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -45,7 +48,12 @@ public void doPost (HttpServletRequest request, HttpServletResponse response) th
             return ;
         }
 
-        comments.add(txt);
+        // Create an entity for the comment and store it
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        Entity ent = new Entity("Task");
+        ent.setProperty("comment", txt);
```

Comment:
Just a small comment, no need to fix: no need to create a variable here: 
```java
ent.setProperty("comment", Entity("Task"));
```

Reply:
not sure I understand. How do I use the ent variable if I do not declare and create it? also the property is set to txt which is a string I used above.